helm-charts/sawmills-remote-operator/tests/image_digest_test.yaml

```
@@ -0,0 +1,14 @@
+suite: image digest rendering
+templates:
+  - deployment.yaml
+tests:
+  - it: should render digest instead of tag
+    set:
+      apiKey: test-key
+      collectorName: sawmills-collector
+      image.tag: 9.9.9
+      image.digest: "sha256:deadbeef"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)